lib/basedriver/driver.js

```diff
@@ -520,7 +530,7 @@ class BaseDriver extends Protocol {
       if (!_.isRegExp(avoidPathRegex)) {
         throw new Error('Proxy avoidance path must be a regular expression');
       }
-      let normalizedUrl = url.replace(/^\/wd\/hub/, '');
+      let normalizedUrl = url.replace(new RegExp(`^${this.basePath}`), '');
```

Comment:
shouldn't we call _.escapeRegexp?

Reply:
same below for other regexps where `this.basePath` is used

Reply:
what is the concern with putting the unescaped string here?

Reply:
i think the worst that could happen is route parsing stuff would stop working. but there are some valid characters like "." that would not be escaped. so yes i will do this.